post-processor/import/post-processor.go

```diff
@@ -224,6 +249,8 @@ func (p *PostProcessor) PostProcess(ctx context.Context, ui packersdk.Ui, artifa
 				},
 			},
 		},
+		Architecture: &p.config.Architecture,
+		BootMode:     &p.config.BootMode,
```

Comment:
I wonder what happened when these were not passed before, do you know ? [Docs](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ec2/import-image.html) are quite vague about it.

Reply:
I was wondering the same. It defaults to `x86_64` and `legacy-bios` if nothing is specified, but I could not quite pinpoint where

Reply:
Cool, thanks for checking ! These sound like the sensible defaults, yes.